examples/toystore/ratelimitpolicy_gateway.yaml

```diff
@@ -1,5 +1,4 @@
----
```

Comment:
I cannot find any reference to this file. It must be a leftover from and old user guide. I vote for deleting the files

Reply:
decided offline to keep them for now.